src/style/app.module.css

```diff
@@ -432,7 +444,7 @@
     --dropdown-hover-color,
     var(--grey-border-box-color)
   ) !important;
-  color: var(--brown-color) !important;
+  color: var(--dark-grey-font) !important;
```

Comment:
Use variable names that describe the function of the color and not the color itself. This will help with adding a Dark Mode feature.
- For example `--search-button-bg` is much better than `--light-blue`